Haltestellenmonitor1-DD.xcodeproj/project.pbxproj

```diff
@@ -1428,7 +1445,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 2;
 				DEVELOPMENT_ASSET_PATHS = "\"WatchMonitor Watch App/Preview Content\"";
-				DEVELOPMENT_TEAM = WGL37RNHJ6;
+				DEVELOPMENT_TEAM = 9FV68KG3UM;
```

Comment:
Please do not change my development team. I will have problems on build for the app store.